migrator/migrator-servlet/src/main/webapp/index.jsp

```diff
@@ -1,23 +1,94 @@
 <!DOCTYPE html>
 <%@ page contentType="text/html;charset=UTF-8" language="java" %>
-<%@ page import="com.google.phonenumbers.HelloWorld" %>
+<%@ page import="com.google.phonenumbers.ServletMain" %>
+<%@ page import="com.google.phonenumbers.migrator.MigrationJob" %>
+<%@ page import="java.io.IOException" %>
+<%
+  final String E164_NUMBERS_LINK = "https://support.twilio.com/hc/en-us/articles/223183008-Formatting-International-Phone-Numbers";
+  final String COUNTRY_CODE_LINK = "https://countrycode.org/";
+  final String DOCUMENTATION_LINK = "./"; // TODO: use README documentation link when uploaded
+%>
 <html>
 <head>
-  <link type="text/css" rel="stylesheet" href="/stylesheets/hello-world.css" />
-  <title>Hello World</title>
+  <link type="text/css" rel="stylesheet" href="/stylesheets/servlet-main.css" />
+  <title>Migrator</title>
 </head>
 <body>
-    <h1>Hello World</h1>
+  <div class="page-heading">
+    <h1>Phone Number Migrator</h1>
+    <p>
+      Enter a path to a text file containing <a href="<%=E164_NUMBERS_LINK%>" target="_blank">E.164 phone numbers</a>
```

Comment:
This might be a bit picky.. :) But could we change this just a little bit. Maybe the first line could be describing what the migrator does on a high level. And then we could have this formatted so that it lists out the two different options more clearly ie (something like this?) 
1. Single Number migration: Takes a single number... etc.
2. File Migration: Takes a file ... etc.

For more information ...

Reply:
Sure, sounds good

Reply:
Revised Introduction Paragraph:
![Screenshot 2020-09-15 at 1 50 54 PM](https://user-images.githubusercontent.com/40521424/93212709-dcb22080-f75a-11ea-9fc8-1e48b706853a.png)